packages/nodes/code-block/src/queries/isSelectionAtStartOfCodeBlock.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  isExpanded,
+  isStartPoint,
+  PlateEditor,
+  Value,
+} from '@udecode/plate-core';
+import { getCodeLineEntry } from './getCodeLineEntry';
+
+/**
+ * Is the selection at the start of the first code line in a code block
+ */
+export const isSelectionAtStartOfCodeBlock = <V extends Value>(
```

Comment:
To be more consistent with `isSelectionAtBlockStart`, could we rename it to `isSelectionAtCodeBlockStart`?